source/deploying-with-ansible/roles/wazuh-opendistro.rst

```diff
@@ -38,6 +38,6 @@ Next, run the ansible playbook: ::
 
   $ ansible-playbook wazuh-elastic.yml -e@vars-production.yml
 
-The example above will install Elasticsearch and set the listen address to: ``10.1.1.10`` using ``vars-production.yml``.
+The example above will install Opendistro and set the listen address to: ``10.1.1.10`` using ``vars-production.yml``.
```

Comment:
```suggestion
The example above will install Open Distro and set the listen address to: ``10.1.1.10`` using ``vars-production.yml``.
```

Reply:
This suggestion is out of date. The requested change is already present:
  - https://github.com/wazuh/wazuh-documentation/pull/4479/files